cluster/gce/gci/configure-helper.sh

```diff
@@ -2148,17 +2168,7 @@ EOF
   fi
   if [[ "${ENABLE_NODE_LOGGING:-}" == "true" ]] && \
      [[ "${LOGGING_DESTINATION:-}" == "gcp" ]]; then
-    setup-addon-manifests "addons" "fluentd-gcp"
```

Comment:
I think this is also not about setting up fluentd parameters, but setting up clusters parameters to use fluentd

Reply:
Moved.